src/io/result_io.jl

```diff
@@ -657,43 +654,46 @@ function load_results(result_loc::String)::ResultSet
     )
 
     outcomes = Dict{Symbol,YAXArray}()
+    outcomes_properties = [:metric_name, :metric_unit, :axes_names, :axes_units]
```

Comment:
Grammatically correct name:

```suggestion
    outcome_properties = [:metric_name, :metric_unit, :axes_names, :axes_units]
```

"properties" signifies the plural